latex/chain.tex

```diff
@@ -528,7 +528,7 @@ \subsection{Operational Certificate Transition}
   \var{vk_{cold}}.
 \end{itemize}
 
-After this, the transition systems updates the operational certificate state by
+After this, the transition system updates the operational certificate state by
```

Comment:
perhaps debatable?

Reply:
I like the singular better as well.  the "transition system" is a collection of "rules" with a given name and type signature. here there is just one "system" doing the updating.